maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ContributionFilesEntity.java

```diff
@@ -0,0 +1,47 @@
+package gov.uk.courtdata.entity;
```

Comment:
Can you move this new entity under the `dces` package in an entity sub-package

i.e. `package gov.uk.courtdata.dces.entity;
`
So new development is in line with the new structure 
https://dsdmoj.atlassian.net/browse/LASB-1971